Extensions/Mongo/Cosmos.DataTransfer.MongoExtension/Settings/MongoSourceSettings.cs

```diff
@@ -5,6 +5,8 @@ public class MongoSourceSettings : MongoBaseSettings
 {
     public string? Collection { get; set; }
 
+    public string? Query { get; set; }
```

Comment:
nit: Let's add some documentation right here that indicates that this can be used either as a JSON string or path to file.